build.sbt

```diff
@@ -2,22 +2,24 @@ name := "play-mockws"
 
 scalaVersion := "2.11.8"
 
-crossScalaVersions := Seq("2.11.8")
+crossScalaVersions := Seq("2.12.1")
```

Comment:
hmm.. I thought we only need to add the "other" scala version in crossScalaVersions, as 2.11.8 is the normal scalaVersion.. ? might be that I'm wrong.